aspnetcore/blazor/components/prerender.md

```diff
@@ -125,6 +128,14 @@ For components embedded into a page or view of a Razor Pages or MVC app, you mus
 </body>
 ```
 
+## Interactive routing and prerendering
+
+Internal navigation for interactive routing doesn't involve requesting new page content from the server. Therefore, prerendering doesn't occur for internal page requests.
+
+The <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service only works on the initial page load and not across enhanced page navigation events. If the app performs a full (non-enhanced) navigation to a page utilizing persistent component state, the persisted state is made available for the app to use when it becomes interactive. But if an interactive circuit has already been established and an enhanced navigation is performed to a page that renders persisted component state, that state isn't made available in the existing circuit. The <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service isn't aware of enhanced navigation, and there's no mechanism to deliver state updates to components that are already running.
+
+A mechanism to deliver state updates for enhanced navigation is under consideration for .NET 9, which is targeted for release in late 2024. For more information, see [[Blazor] Support persistent component state across enhanced page navigations (dotnet/aspnetcore #51584)](https://github.com/dotnet/aspnetcore/issues/51584). For more information on enhanced navigation, see <xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling>.
```

Comment:
We may want to move these two paragraphs to a separate section, since they apply to cases where there's a static (non-interactive) router.

Reply:
Yes, I agree ... I think the problem is that the persist component state service is only discussed in light of maintaining state ***for prerendering*** (i.e., in this article). It's broader than that, so these two paragraphs can go where the persist component state service is described elsewhere and then the use of the service for prerendering situations can be cross-linked from this article to that new location ......

I just need to see ***where*** that the service guidance should live.